zproject/urls.py

```diff
@@ -329,6 +330,8 @@
     # Endpoints for syncing drafts.
     rest_path("drafts", GET=fetch_drafts, POST=create_drafts),
     rest_path("drafts/<int:draft_id>", PATCH=edit_draft, DELETE=delete_draft),
+    rest_path("saved_replies", GET=get_saved_replies, POST=create_saved_reply),
+    rest_path("saved_replies/<int:saved_reply_id>", DELETE=delete_saved_reply),
```

Comment:
Make sure to add a separate comment to your changes from `# Endpoints for syncing drafts.`